test/jetstream_test.go

```diff
@@ -2182,6 +2182,161 @@ func TestJetStreamWildcardSubjectFiltering(t *testing.T) {
 	}
 }
 
+func TestJetStreamWildcardSubjectFilteringEphemeralConsumer(t *testing.T) {
+	subjects := []string{"js.test.one", "js.test.two", "js.test.three"}
```

Comment:
Changing this into `[]string{"js.test.*"}` would also make it possible to use a wildcard in the FilterSubject and the test would be the same.